specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/examples/FrontdoorCreate.json

```diff
@@ -83,7 +83,7 @@
                   "priority": 1
                 },
                 {
-                  "privateLinkResourceId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/privateLinkServices/pls1",
+                  "privateLinkAlias": "SQLSERVER.d84e61f0-0870-4d24-9746-7438fa0019d1.westus2.azure.privatelinkservice",
```

Comment:
>SQLSERVER.d84e61f0-0870-4d24-9746-7438fa0019d1.westus2.azure.privatelinkservice [](start = 39, length = 79)

this this a real alias?

Reply:
Please ensure that any example private link aliases do not actually exist in production

Reply:
This is not a real alias, I generated it as an example